src/Common/src/Interop/Unix/Interop.Errors.cs

```diff
@@ -61,6 +61,7 @@ internal enum Error
         ENETUNREACH      = 0x10028,           // Network unreachable.
         ENFILE           = 0x10029,           // Too many files open in system.
         ENOBUFS          = 0x1002A,           // No buffer space available.
+        ENODATA          = 0x1002B,           // No message is available on the STREAM head read queue.
```

Comment:
These four values were just explicitly removed in https://github.com/dotnet/corefx/pull/2864.  Is this just a copy-and-paste mistake, or are they actually needed?